core/src/main/java/io/questdb/cutlass/text/CairoTextWriter.java

```diff
@@ -175,11 +193,24 @@ public void onFieldsPartitioned(long line, ObjList<DirectByteCharSequence> value
                 if (onField(line, dbcs, w, i)) return;
             }
             w.append();
+            checkMaxAndCommitLag();
         } catch (Exception e) {
             logError(line, timestampIndex, dbcs);
         }
     }
 
+    private void checkMaxAndCommitLag() {
+        if (writer != null) {
+            if (durable) {
+                assert false;
```

Comment:
Why assert, is it impossible to have durable as false?

Reply:
in line 104 there is a commit method:

```
public void commit() {
        if (writer != null) {
            if (durable) {
                assert false;
            } else {
                writer.commit();
            }
        }
    }
```
I suppose `durable` is not supported?